pytest/lib/branches.py

```
@@ -16,14 +16,15 @@ def compile_binary(branch):
     # TODO: download pre-compiled binary from github for beta/stable?
     prev_branch = current_branch()
     stash_output = subprocess.check_output(['git', 'stash'])
-    subprocess.call(['git', 'checkout', branch])
-    subprocess.call(['cargo', 'build', '-p', 'near'])
-    subprocess.call(['cargo', 'build', '-p', 'state-viewer'])
+    subprocess.check_output(['git', 'checkout', branch])
```

call fails silently, checkout_output raise issue when command fail